packages/council-proposals/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@elementfi/council-proposals",
+  "version": "1.0.2",
+  "description": "",
+  "dependencies": {
+    "@elementfi/elf-council-tokenlist": "^1.0.2",
+    "@elementfi/elf-council-typechain": "^1.0.2",
```

Comment:
hmm, looks like we'll need to migrate these projects over as well:

https://github.com/element-fi/elf-council-typechain
https://github.com/element-fi/elf-council-tokenlist

They're currently published to npm so everything still works here, but let's do this in a follow up PR